src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -1370,6 +1371,16 @@ BoundExpression bindDefaultArgument(SyntaxNode syntax, ParameterSymbol parameter
                     var memberName = containingMember.GetMemberCallerName();
                     defaultValue = new BoundLiteral(syntax, ConstantValue.Create(memberName), Compilation.GetSpecialType(SpecialType.System_String)) { WasCompilerGenerated = true };
                 }
+                else if (callerSourceLocation is object && getArgumentIndex(parameter.CallerArgumentExpressionParameterIndex, argsToParamsOpt) is int argumentIndex &&
+                    argumentIndex > -1 && argumentIndex < argumentsCount)
+                {
+                    CheckFeatureAvailability(syntax, MessageID.IDS_FeatureCallerArgumentExpression, diagnostics);
+                    // PROTOTYPE(caller-expr): Do we need to support VB?
+                    // PROTOTYPE(caller-expr): Do we need to add a warning/error for pre-C# 10 using this feature?
```

Comment:
>// PROTOTYPE(caller-expr): Do we need to add a warning/error for pre-C# 10 using this feature? [](start = 20, length = 94)

Remove the prototype comment about the version check?

Reply:
Will do with a general cleanup tomorrow.

Reply:
Done.